tests/lsdb/loaders/hipscat/test_read_hipscat.py

```diff
@@ -94,11 +99,6 @@ def test_catalog_without_margin_is_none(small_sky_xmatch_dir):
     assert catalog.margin is None
 
 
-def test_catalog_with_wrong_margin_args(small_sky_xmatch_dir):
-    with pytest.raises(ValueError, match="must be of type"):
-        lsdb.read_hipscat(small_sky_xmatch_dir, margin_cache=1)
-
-
```

Comment:
Why do you delete this one?

Reply:
I removed the type checking so this error isn't thrown.